pkg/kubelet/images/image_gc_manager_test.go

```diff
@@ -193,6 +193,25 @@ func TestDeleteUnusedImagesExemptSandboxImage(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestDeletePinnedImage(t *testing.T) {
```

Comment:
I'd recommend adding cases for:
- List with no pinned images
- List with only pinned images
- List with mixed images

Reply:
added!